mindpype/kernels/pad.py

```diff
@@ -102,11 +129,14 @@ def add_to_graph(cls, graph, inA, outA, pad_width = None, mode = 'constant',
         graph : Graph
             Graph that the kernel should be added to
         inA : Tensor
-            Input trial data (n_channels, n_samples) or (n_trials, n_channels, n_samples)
+            Input data (n_channels, n_samples) or (n_trials, n_channels, n_samples)
         outA : Tensor
-            Output trial data (n_channels, n_samples) or (n_trials, n_channels, n_samples)
+            Output data (n_channels, n_samples) or (n_trials, n_channels, n_samples)
         pad_width : int or sequence of ints, optional
-            Number of values padded to the edges of each axis. ((before_1, after_1), ... (before_N, after_N)) unique pad widths for each axis. ((before, after),) yields same before and after pad for each axis. (pad,) or int is a shortcut for before = after = pad width for all axes. Default is None, in which case no padding is added.
+            Number of values padded to the edges of each axis. ((before_1, after_1), ... (before_N, after_N)) 
```

Comment:
This looks like some of the parameter descriptions are copied from the numpy.pad docstring. Let's simplify these and add an explicit reference to the numpy documentation instead. See transpose for an example.